### ISSUE TITLE
Unable to Deploy Firebase functions after upgrading to SDK 11.0.1 or firebase-admin v11.0.1

### ISSUE THREAD

**Issue by fnisarm** [firebase/firebase-admin-node#1915]
* Operating System version: Windows 10
  * Firebase SDK version: 11.0.1
  * Firebase Product: functions
  * Node.js version: v16.11.0
  * NPM version: 8.0.0

### [REQUIRED] Step 3: Describe the problem
Unable to Deploy Firebase Functions after recent upgrade to SDK 11

#### Steps to reproduce:
Firebase Deploy or npm install firebase-admin connect-session-firebase --save

#### Relevant Code:

```

Build failed: npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: connect-session-firebase@5.13.0
npm ERR! Found: firebase-admin@11.0.1
npm ERR! node_modules/firebase-admin
npm ERR!   firebase-admin@"^11.0.1" from the root project
npm ERR!   peer firebase-admin@"^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0" from firebase-functions@3.23.0
npm ERR!   node_modules/firebase-functions
npm ERR!     firebase-functions@"^3.23.0" from the root project
npm ERR!     peer firebase-functions@">=2.0.0" from firebase-functions-test@0.2.1
npm ERR!     node_modules/firebase-functions-test
npm ERR!       dev firebase-functions-test@"^0.2.0" from the root project
npm ERR!   1 more (firebase-functions-test)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer firebase-admin@"~5.13.0" from connect-session-firebase@5.13.0
npm ERR! node_modules/connect-session-firebase
npm ERR!   connect-session-firebase@"^5.13.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: firebase-admin@5.13.1
npm ERR! node_modules/firebase-admin
npm ERR!   peer firebase-admin@"~5.13.0" from connect-session-firebase@5.13.0
npm ERR!   node_modules/connect-session-firebase
npm ERR!     connect-session-firebase@"^5.13.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

```

**Comment by lahirumaramba** [comment]
It looks like `connect-session-firebase` still depends on a legacy version of `firebase-admin`.
I believe this should be fixed in the `connect-session-firebase` package. https://github.com/benweier/connect-session-firebase/issues/126

**Comment by benweier** [comment]
In case anyone else finds this issue, I have published `connect-session-firebase@11.0.0` to address the outdated dependencies.
